composer.json

```diff
@@ -4,8 +4,8 @@
     "description": "Google Analytics integration for Neos CMS",
     "license": "GPL-3.0-or-later",
     "require": {
-        "neos/neos": "~4.3 || ~5.0 || dev-master",
-        "neos/fusion-afx": "^1.2",
+        "neos/neos": "^7.0 || dev-master",
+        "neos/fusion-afx": "^7.0",
```

Comment:
```suggestion
```

Reply:
Think we don't need this explicitly

Reply:
We do, the code uses AFX… Relying on it "to be there already" is doubtful.